clj-refactor.el

```diff
@@ -2981,8 +3028,8 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-find-usages"
         (when (looking-at-p "(\\s-*def")
           (re-search-forward name)
           (paredit-backward))
-        (when (looking-at (format "\\(.+/\\)?\\(%s\\)" name))
-          (replace-match (format "\\1%s" new-name))))
+        (when (re-search-forward (format "\\(.+/\\)?\\(%s\\)" name) search-limit t)
```

Comment:
This is what we had back in the day, but we had to change it to the current version because it caused problems.  I think this might introduce a regression.  You should walk the git history to find out exactly why we changed it the last time.